src/fluxgui/settings.py

```diff
@@ -158,7 +159,8 @@ def _create_autostarter(self):
             #create autostart entry
             starter_item = DesktopEntry(autostart_file)
             starter_item.set('Name', 'f.lux indicator applet')
-            starter_item.set('Exec', 'fluxgui')
+            starter_item.set('Comment', 'Better lighting for your computer')
+            starter_item.set('Exec', os.path.realpath(sys.argv[0]))
```

Comment:
I like this idea, but I think there might be a simpler and more robust solution: instead of hardcoding the current path to `fluxgui` when the .autostart file is created, we hard code it to run `fluxgui` in the user's default shell, which will incorporate any path modifications that make the local install work when they run it from a terminal.

I.e., we use `'%s -c fluxgui'' % os.getenv('SHELL')` for the exec value. E.g. this is `/bin/zsh -c fluxgui` for ZSH users. I checked and all of `sh`, `bash`, `zsh`, `csh`, and `ksh` support the `-c <cmd>` option.

This is simpler because we don't have to change the logic for spawning `xflux` (your second commit), and more robust because the .autostart file won't break if the user later changes where `fluxgui` is installed (the .autostart file only gets updated if they delete it, and it's not part of the `installed.txt` list). A downside compared to your approach is that this could break if the user changed their shell and then installed `fluxgui` to a different local location that was only on the `PATH` for their new shell, but that seems unlikely.

What do you think?

Reply:
I considered it (both manually for my autostarts in general and for this). I think I prefer the technique in this PR: guaranteed to work first time (less fiddling/debugging); no shell spawned; also works when launched from outside `$PATH`.

In contrast, the shell method is not guaranteed to work (we have to worry about login vs non-login shells & various startup scripts, which would have still left it broken for me - this kind of thing is no fun at all). I do many more `$PATH` manipulations than moves of already-installed binaries, including sometimes removing things from my `$PATH`. I would guess this is relatively uncommon, but still more common than moving binaries. Note that the autostart is deleted/re-installed also when the checkbox is toggled.

That said, I take your point that some people will move the binary to somewhere else in their `$PATH` and expect it to still work. If you prefer I can modify the PR, but I'd suggest keeping the fallback behaviour for `xflux` in addition to trying the shell (since multiple methods are possible).